spec/factories/plant_line.rb

```diff
@@ -3,10 +3,7 @@
     sequence(:plant_line_name) {|n| "#{Faker::Lorem.characters(5)}_#{n}"}
     common_name { Faker::Lorem.word }
     previous_line_name { Faker::Lorem.word }
-    date_entered { Faker::Date.backward }
-    data_owned_by { Faker::Company.name }
-    comments { Faker::Lorem.sentence }
-    data_provenance { Faker::Lorem.sentence }
     taxonomy_term
+    instance_eval &AnnotableFactory.annotated
```

Comment:
Not saying this is bad solution, but the same should be possible with a `trait`:

```ruby
FactoryGirl.define do
  trait :annotable do
     date_entered { ... }
  end

  factory :plant_line do
    annotable
  end
end
```